diff -Nur incron-0.5.10.orig/appargs.cpp incron-0.5.10/appargs.cpp
--- incron-0.5.10.orig/appargs.cpp	2012-04-06 16:19:27.000000000 -0600
+++ incron-0.5.10/appargs.cpp	2012-08-26 10:54:34.587894586 -0600
@@ -25,6 +25,7 @@
 
 #include <cstdio>
 #include <cstring>
+#include <cstdio>
 
 #include "strtok.h"
 
diff -Nur incron-0.5.10.orig/appinst.h incron-0.5.10/appinst.h
--- incron-0.5.10.orig/appinst.h	2012-04-06 16:19:27.000000000 -0600
+++ incron-0.5.10/appinst.h	2012-08-26 10:54:34.588894591 -0600
@@ -23,6 +23,7 @@
 
 
 #include <string>
+#include <cstdio>
 
 
 #define APPLOCK_BASEDIR "/var/run"
diff -Nur incron-0.5.10.orig/icd-main.cpp incron-0.5.10/icd-main.cpp
--- incron-0.5.10.orig/icd-main.cpp	2012-04-06 16:19:27.000000000 -0600
+++ incron-0.5.10/icd-main.cpp	2012-08-26 10:54:15.329794395 -0600
@@ -28,6 +28,7 @@
 #include <sys/stat.h>
 #include <cstdio>
 #include <cstring>
+#include <unistd.h>
 
 #include "inotify-cxx.h"
 #include "appinst.h"
diff -Nur incron-0.5.10.orig/incroncfg.h incron-0.5.10/incroncfg.h
--- incron-0.5.10.orig/incroncfg.h	2012-04-06 16:19:27.000000000 -0600
+++ incron-0.5.10/incroncfg.h	2012-08-26 10:54:34.589894598 -0600
@@ -19,6 +19,7 @@
 
 
 #include <cstring>
+#include <cstdio>
 #include <map>
 
 /// Configuration class.
diff -Nur incron-0.5.10.orig/inotify-cxx.h incron-0.5.10/inotify-cxx.h
--- incron-0.5.10.orig/inotify-cxx.h	2012-04-06 16:19:27.000000000 -0600
+++ incron-0.5.10/inotify-cxx.h	2012-08-26 10:54:34.589894598 -0600
@@ -28,6 +28,7 @@
 
 #include <stdint.h>
 #include <string>
+#include <cstdio>
 #include <deque>
 #include <map>
 
